### DISCUSSION TITLE
Agents are not pinging.

### DISCUSSION THREAD

**Discussion by bhakarswayamavarapu** [Xceptance/XLT#476] [General]
Hi team,

Today i was trying to add 3 more agent machines to run the load test. I am able to start the agent controller in putty and output was "picked-up". But the agents are not pinging from the master controller.

Screenshot:
![image](https://github.com/Xceptance/XLT/assets/140174174/01083ad5-2a91-481e-aafd-038af4445f64)
Screenshot from mastercontroller:
![image](https://github.com/Xceptance/XLT/assets/140174174/27c73d3b-7b86-4443-b10b-0daf05c5dc3c)

Can you please suggest?

Thanks!

**Comment by rschwietzke** [comment]
Did you open the firewall for that port?

**Reply by bhakarswayamavarapu** [reply]
Yes, to open the port i have used the highlighted command.
![image](https://github.com/Xceptance/XLT/assets/140174174/4ccc0d8e-ac97-4f02-ad3e-3f9a0bb24c56)

Thanks!

**Reply by bhakarswayamavarapu** [reply]
Hi @rschwietzke,
For your information below is the error log
![image](https://github.com/Xceptance/XLT/assets/140174174/063396a2-8fb2-4c1c-865f-f9ad6cd45fb0)
Thanks!

**Comment by rschwietzke** [comment]
We cannot diagnose anything here. When I test the port with https://www.yougetsignal.com/tools/open-ports/ it says closed. Also ensure that your mastercontroller machine is permitted to communicate out on that port as well. You probably want to open input and output.
